src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -480,6 +480,21 @@ internal static bool VersionLessThanOrEquals(string a, string b)
             return SimpleVersion.Parse(a) <= SimpleVersion.Parse(b);
         }
 
+        internal static string GetTargetFrameworkIdentifier(string tfm)
+        {
+            return NuGetFramework.Value.GetTargetFrameworkIdentifier(tfm);
+        }
+
+        internal static string GetTargetFrameworkVersion(string tfm)
+        {
+            return NuGetFramework.Value.GetTargetFrameworkVersion(tfm);
+        }
+
+        internal static bool IsTargetFrameworkCompatible(string tfm1, string tfm2)
```

Comment:
nit: Maybe use the descriptive names?
Compatibility is a one-way relationship the order matters. 

Call the target and candidate, similar to what the NuGet API calls them?

Reply:
Sure, just made that change. (Had to force push since the PR didn't pick it up when I updated the branch the first time).